app/models/header_lookup.rb

```diff
@@ -135,11 +162,17 @@ def strip_accents(term="")
   def read_memoized_headers(filepath)
     begin
       file = File.read(filepath)
+      # in read / write mode
+      # file = File.open(filepath, "w+")
       # eval will interpret a ruby string passed as an argument
       # in this case it constructs a hash from the file
       eval(file)
     rescue Errno::ENOENT
       return {}
     end
   end
+
+  def downcase_and_remove_non_alphanumeric(str)
+    str.downcase.gsub(/[^\w\s]/, '')
```

Comment:
Because this field is ultimately free text it might have incorrect casing, but what was the issue with non-alphanumeric? Also we can add whitespace trimming on either side.

Reply:
It was an attempt to try and normalize the terms that are being iterated over to find matches. I don't think it's really working though. 

After coming back to it with a clearer head I can think of a few examples where this would fail with the current replacement character. Using a space as the replacement character might work, but I think the bigger question is should we bother with trying to normalize at all?

Reply:
Let's get a sense for how that field looks like in staging/production and then decide? I don't think we can go wrong with stripping/trimming and downcasing just for comparison sake, though e.g., "Napoleon " and "napoleon" should be treated the same.